### PR TITLE
Updated IsJson boolean method to ValidateJson void

### PR DESCRIPTION
**Pull request by hakimms**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs [hunk 1]
```diff
@@ -108,19 +108,16 @@ internal static HttpResponseMessage HttpJsonResponse(AuthenticationEventJsonElem
...
         /// <param name="input">The input.</param>
-        /// <returns>
-        ///   <c>true</c> if the specified input is json; otherwise, <c>false</c>.</returns>
-        internal static bool IsJson(string input)
+        internal static void ValidateJson(string input)
         {
             if (string.IsNullOrEmpty(input))
             {
-                return false;
+                throw new Exception();
```
Comment (HarmanDhunna) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs:115]: Should this be JsonException?

Comment (hakimms) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs]: Updated this and added a message that Json passed is empty

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/PayloadTests.cs [hunk 2]
```diff
@@ -64,7 +64,7 @@ public async Task Tests(TestTypes testType)
                 case TestTypes.NoAction:
                     return (Payloads.TokenIssuanceStart.TokenIssuanceStart.NoActionResponse, @"{'errors':['No Actions Found. Please supply atleast one action.']}", HttpStatusCode.InternalServerError);
                 case TestTypes.Empty:
-                    return (string.Empty, @"{'errors':['Return type is invalid, please return either an AuthEventResponse, HttpResponse, HttpResponseMessage or string in your function return.']}", HttpStatusCode.InternalServerError);
+                    return (string.Empty, @"{'errors':['Return type is invalid, please return either an AuthEventResponse, HttpResponse, HttpResponseMessage or string in your function return: Json passed is empty.']}", HttpStatusCode.InternalServerError);
```
Comment (HarmanDhunna) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/PayloadTests.cs:67]: JSON is empty or null

**Review by HarmanDhunna — APPROVED** [review]

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/PayloadTests.cs (hunk 2 above)
Comment (hakimms) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/PayloadTests.cs]: Updated string

**Review by HarmanDhunna — APPROVED** [review]

**Pull request merged**